sinks/v1/grpc-consumer/src/main/java/com/google/cloud/trace/grpc/v1/GrpcTraceConsumer.java

```diff
@@ -63,9 +59,14 @@ public void receive(Traces traces) {
   }
 
   /**
-   * Closes the resources held by this trace consumer.
+   * Creates a trace consumer that sends trace messages to the Stackdriver Trace API via gRPC.
+   *
+   * @param apiHost     a string containing the API host name.
+   * @param credentials a credentials used to authenticate API calls.
    */
-  public void close() {
-    managedChannel.shutdown();
+  public static GrpcTraceConsumer create(String apiHost, Credentials credentials) {
```

Comment:
I was thinking to move this more towards moving construction of the stub out of the class altogether.